CMakeLists.txt

```diff
@@ -122,7 +130,6 @@ if(TRAVIS_CI_BUILD)
 endif()
 
 if(CMAKE_COMPILER_IS_GNUCC)
-  include(CheckCCompilerFlag)
   check_c_compiler_flag(-Og HAS_OG_FLAG)
 else()
   set(HAS_OG_FLAG 0)
```

Comment:
BTW, the else side was intentional as well.  CMake is starting to warn about using unassigned variables, so that was meant to keep it from screaming at you in those versions.